docs/DeveloperGuide.md

```diff
@@ -185,13 +195,18 @@ containing flashcards that have the same tag only (to be implemented...)
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                 | I want to …​                        | So that I can…​                              |
-| -------- | ------------------------------------------ | -------------------------------------- | ----------------------------------------------- |
-| `* * *`  | new user                                   | view valid commands                    | remember how to use the App                     |
-| `* * *`  | user                                       | view a flashcard                       |                                                 |
-| `* * *`  | user                                       | see the answer to a flashcard          | check if I answered correctly                   |
-| `* * *`  | user                                       | start a practice run of all flashcards | practice all flashcards in a single session     |
-| `* *`    | user                                       | view all flashcards                    | study the flashcards before a session           |
+| Priority | As a …​         | I want to …​                            | So that I can…​                              |
+| -------- | ------------------ | ------------------------------------------ | ----------------------------------------------- |
+| `* * *`  | new user           | view valid commands                        | remember how to use the Weeblingo               |
```

Comment:
It would look fancier if we add descriptive words before `user`s; E.g. as a dilligent user/learner or sth?

Reply:
Imeanlike sure but maybe sometime later. This isn't gonna make or break it for us.